README.md

```diff
@@ -78,47 +78,229 @@ Here's what each of these steps does:
+   - Update either the PR or the person requesting the run to let them know the run has started.
+   - Go to next run.
+ - Preparing
+   - Log onto appropriate box and connect to multiplexer.
+   - Switch to the `master` multiplexer window.
+   - If preparation is ongoing, go to next run.
+   - If preparation failed, fix it. Known errors:
+     - "missing sha for ..." - remove the referenced repository from `gh-apps.txt`
+       and `gh-candidates.txt` (may be present in one or both). Make the same
+       change locally and make a PR against cargobomb. Use
+       `cargo run --release -- delete-all-target-dirs --ex EX_NAME` and
+       `cargo run --release -- delete-ex --ex EX_NAME`, then jump to start of 'Pending'.
+   - Switch to the `tc1` multiplexer window.
+   - Run `cargo run --release -- run-tc --ex EX_NAME EX_START`.
+   - Switch to the `tc2` multiplexer window.
+   - Run `cargo run --release -- run-tc --ex EX_NAME EX_START`.
+   - Go to next run.
+ - Running
+   - Log onto appropriate box and connect to multiplexer.
+   - Switch to the `master` multiplexer window.
+   - Run `docker ps`. If any container has been running for more than 30min (may
+     need to follow these steps more than once):
+     - Take solace in us someday fixing this for good with docker limits.
+       TODO: actually fix. Seems to only be a problem on prod with pleingres,
+       our existing limits should catch it.
+     - Run `docker top CONTAINER_ID`.
+     - If there's no mention of pleingres, raise an issue with the output of
+       the previous `docker top` command.
+     - The process at the bottom of the list is the lowest in the process tree,
+       and should have a value in the `TIME` column of >30min. Find the value in
+       the `PID` column and run `kill PID`.
+     - Wait a few seconds, then check the container has now exited.
+   - If the run is ongoing in either the `tc1` or `tc2` multiplexer
+     windows, go to next run.
+   - Switch to the `master` multiplexer window.
+   - Run `du -sh work/ex/EX_NAME`, output should be <2GB. If not:
+     - Run `find work/ex/EX_NAME -type f -size +100M | xargs du -sh`,
+       there will likely only be a couple of files listed and they
+       should be in the `res` directory (TODO: blacklist pleingres
+       as the main culprit here once it's possible, and update
+       these instructions to suggest adding things to the blacklist).
+     - For each file found, run `truncate --size='<100M' FILE`.
+     - Check `du -sh work/ex/EX_NAME` is now an appropriate size.
+   - Run `cargo run --release -- publish-report --ex EX_NAME s3://cargobomb-reports/EX_NAME`.
+   - Change status to 'Uploading'.
+   - (optional but much appreciated: come back to this run in 30mins
+     as the upload will be complete)
+   - Go to next run.
+ - Uploading
+   - Switch to the `master` multiplexer window.
+   - If the upload is ongoing, go to the next run.
+   - Run `cargo run --release -- delete-all-target-dirs --ex EX_NAME`.
```

Comment:
We can do this in parallel to the uploading above. The target dirs don't contain the results.